Fw/Tlm/docs/sdd.md

```diff
@@ -17,16 +17,38 @@ The `Fw::Tlm` port has the following port diagram:
 
 #### 2.1.2 Serializables
 
-##### 2.1.2.1 Fw::ComPacket 
+##### 2.1.2.1 Fw::ComBuffer
 
-The `Fw::TlmPacket` class is a packet class derived from [`Fw::ComPacket`](../../Com/docs/sdd.html) that provides methods for encoding a telemetry packet. 
+The `Fw::TlmBuffer` class contains a buffer that holds the serialized value of a telemetry channel. This buffer is passed as an argument to the `Fw::Tlm` port.
 
-##### 2.1.2.2 Fw::ComBuffer
+##### 2.1.2.2 Fw::ComPacket 
 
-The `Fw::TlmBuffer` class contains a buffer that holds the serialized value of a telemetry channel. This buffer is passed as an argument to the `Fw::Tlm` port.
+The `Fw::TlmPacket` class is a packet class derived from [`Fw::ComPacket`](../../Com/docs/sdd.md) that provides methods for encoding a telemetry packet. It contains an internal `Fw:ComBuffer` that holds serialized channel values.
+
+To fill a packet with telemetry values, do the following.
+
+1\. Instantiate an instance of the `TlmPacket` class.
```

Comment:
The changes to the SDD look good. I fixed a broken link.

It looks like you have suppressed the line-byline numbering by writing 1\. Is this intentional? I think it's easier to read if each numbered item starts on a new line.

Reply:
At a certain point I got tired fighting the auto-numbering. I couldn't predict when it would restart, so I manually number.

Reply:
Manual numbering is fine, but can we insert a blank line between the enumerated items? Otherwise they all get mashed together in a single paragraph with no line breaks.

Reply:
Yes - I'll fix that. I didn't realize they were mashing together.

Reply:
OK, sounds good.

Reply:
@bocchino Fixed the numbering. Should be ready.

Reply:
Looks good. Again there appears to be some bug in the GitHub PR viewer (at least in my browser) but when I click on "edit file" I see the changes.